portal/static/js/src/modules/TnthAjax.js

```diff
@@ -22,8 +22,9 @@ export default { /*global $ */
     },
     "sendRequest": function(url, method, userId, params, callback) {
         if (!url) { return false; }
-        var REQUEST_TIMEOUT_INTERVAL = 5000;
-        var defaultParams = {type: method ? method : "GET", url: url, attempts: 0, max_attempts: MAX_ATTEMPTS, contentType: "application/json; charset=utf-8", dataType: "json", sync: false, timeout: 5000, data: null, useWorker: false, async: true};
+        var REQUEST_TIMEOUT_INTERVAL = 10 * 1000; // default timed out at 10 seconds
+        var RETRY_INTERVAL = 5000; // wait 5 seconds before retry
```

Comment:
does it make sense to retry after 5 seconds, when a request is given 10 to timeout?
seems like this might lead to stacking requests?  i'm not sure, but using the same interval for timeout and retries makes sense to me.

Reply:
thank you @pbugni that makes sense. I fixed it.